subtrack.MAUI/Pages/Index.razor

```diff
@@ -36,7 +36,7 @@
                             <i class="fa fa-repeat" aria-hidden="true"></i>
                         }
                     </div>
-                    <div class="text-end">@GetDueDaysText(sub.DueDays)</div>
+                    <div class=@CssUtil.GetDueClass(sub.DueDays)>@GetDueDaysText(sub.DueDays)</div>
```

Comment:
Did you try running this? I think we still need the text-end because it aligns the text

Reply:
![home](https://github.com/Code2Gether-Discord/subtrack/assets/53407843/72781bfd-8484-40d8-ac64-5a4530a3fa66)
![detail](https://github.com/Code2Gether-Discord/subtrack/assets/53407843/8b681e56-4c81-4fed-8363-1bb7f83f2c30)
I am not sure why but it seems to have retained the position of the text.

Reply:
> ![home](https://github.com/Code2Gether-Discord/subtrack/assets/53407843/72781bfd-8484-40d8-ac64-5a4530a3fa66)
> ![detail](https://github.com/Code2Gether-Discord/subtrack/assets/53407843/8b681e56-4c81-4fed-8363-1bb7f83f2c30)
> I am not sure why but it seems to have retained the position of the text.

Could be bug related. Before you remove it check the git history on that line, alternatively just keep it so you dont need to waste time

Reply:
> ![home](https://github.com/Code2Gether-Discord/subtrack/assets/53407843/72781bfd-8484-40d8-ac64-5a4530a3fa66)
> ![detail](https://github.com/Code2Gether-Discord/subtrack/assets/53407843/8b681e56-4c81-4fed-8363-1bb7f83f2c30)
> I am not sure why but it seems to have retained the position of the text.

Btw you should really pay your paramount subscription